VS_Project/Pokerapp/Pokerapp/spelers-uitschakelen.cshtml

```diff
@@ -0,0 +1,93 @@
+﻿@using WebMatrix.Data
+
+@{
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    //Insert Nick into Players
+    string name = "Nick";
+    int room_code = 1234567;
+    int tbl_nr = 2;
+    //int rebuy = 1;
+    //string insert = "INSERT INTO Players (Name, Room_Code, Tbl_nr, Rebuy) VALUES (@0, @1, @2, @3)";
+    //db.Execute(insert, name, room_code, tbl_nr, rebuy);
+
+    int maxrebuy = 3;
+    string player = (string)db.QueryValue("SELECT Name FROM Players WHERE Name = @0 AND Room_Code = @1 AND Tbl_nr = @2", name, room_code, tbl_nr);
+    int rebuyvalue = (int) db.QueryValue("SELECT Rebuy FROM Players WHERE Name = @0 AND Room_Code = @1", name, room_code);
+
+    if (IsPost && !Request.Form["rebuy"].IsEmpty() && rebuyvalue < maxrebuy)
+    {
+        var rebuyplayer = db.Execute("UPDATE Players SET Rebuy = @0, Tbl_nr = @1 WHERE Name = @2 AND Room_Code = @3", rebuyvalue + 1, tbl_nr, name, room_code);
+    }
+
+    if (IsPost && !Request.Form["rebuy"].IsEmpty() && rebuyvalue == maxrebuy - 1 || rebuyvalue == maxrebuy)
+    {
+    <!-- Modal -->
+    <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Player @player has reached the maximum amount of @maxrebuy rebuys</h5>
+                </div>
+                <div class="modal-body">
+                    <form method="post" name="optionsform" class="optionsform">
+                        <div>
+                            <button value="delete" name="delete" type="submit">Delete</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    }
+
+    if (IsPost && !Request.Form["delete"].IsEmpty())
+    {
+        var deleteplayer = db.Execute("UPDATE Players SET Tbl_nr = NULL WHERE Name = @0 AND Room_Code = @1 AND Tbl_nr = @2", name, room_code, tbl_nr);
```

Comment:
Je vult hier een variabele (**var**) met een method die void returnt. Wat verwacht je te krijgen uit deze line aan code? dat je iets doet of dat je iets op slaat in je variabele?